forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -1087,6 +1096,24 @@ export const RecordsLog = ({
                 fullWidth
               >
                 {recordsDownloadList.map((item, index) => {
+                  if (item.id === 1 && item.disabled) {
+                    return (
+                    <Tooltip title={<div style={{fontSize: "11px"}}>File conversion and deduplication in progress</div>} key={item.id}>
+                      <div>
+                        <MenuItem
+                          className="download-menu-item"
+                          key={item.id}
```

Comment:
can you try combining this with the bottom code block? Tooltip should be able to be disabled using disableHoverListener https://mui.com/material-ui/api/tooltip/#Tooltip-prop-disableHoverListener

Reply:
Attempted to implement this but the Menu Items became buggy and control over the tooltip did not seem to work well. Left this code as is.